src/java.base/share/classes/java/lang/Float16.java

```diff
@@ -48,6 +51,16 @@
  * <p>This is a <a href="https://openjdk.org/jeps/401">primitive value class</a> and its objects are
  * identity-less non-nullable value objects.
  *
+ * @apiNote
+ * The methods in this class generally have analogous methods in
+ * either {@link Float}/{@link Double} or {@link Math}/{@link
+ * StrictMath}. Unless otherwise specified, the handling of special
+ * floating-point values such as {@linkplain #isNaN(Float16) NaN}
+ * values, {@linkplain #isInfinite(Float16) infinities}, and signed
+ * zeros of methods in this class is wholly analogous to the handling
+ * of equivalent cases by methods in {@code Float}, {@code Double},
+ * {@code Math}, etc.
+ *
```

Comment:
In the mainline JDK, we've long since stopped adding new `@author` tags. (I've actually removed some of my `@author` tags in some test code.) In preparation for a future where Float16 is in the mainline JDK, I suggest this information be recorded in some other fashion, at least when integrated into the mainline.

The `/contributor ` Skara command can be used to record multiple co-authors of the code in a PR. That is one way this information could be preserved when integrated into the mainline.

Reply:
Hi @jatin-bhateja,
With the latest commit, I think the spec is close to the spec that would be acceptable to include in mainline. Please confirm you're okay with this state of the PR being integrated. Afterward, work can continue refining a few areas binary <-> decimal spec and implementation, fma internal comment and/or implementation, etc. Thanks.